index.bs

```diff
@@ -157,13 +158,16 @@ validation of the user's location. For example, if the end user is connected thr
 field associated with geo IP information can be compared with magnetometer readings at real
 location, therefore, tell whether user is using VPN or not.
 
-To mitigate these Magnetometer specific threats, user agents should
+Uncalibrated magnetometer readings could be affected by magnetized object nearby, such as jewelry,
```

Comment:
[nit]
Uncalibrated magnetometer readings could be affected by magnetized objects nearby, such as jewelry, 
thereby exposing information that might be used for [=keystroke monitoring=].